packages/wrangler/src/reporting.ts

```diff
@@ -30,6 +30,15 @@ export function initReporting() {
       // from their worker during development
       return null;
     },
+    beforeSend: (event) => {
+      return !event.exception?.values?.some((value) =>
```

Comment:
We should definitely have a comment here describing what we are doing and why.

Reply:
Done.